src/firstvoices/backend/models/part_of_speech.py

```diff
@@ -6,24 +6,36 @@
 from .base import BaseModel
 
 
+class ParentManager(models.Manager):
+    """Manager to convert foreign key relationship to natural keys for fixtures to load correctly."""
+
+    def get_by_natural_key(self, title):
+        return self.get(title=title)
+
+
 class PartOfSpeech(BaseModel):
     """Model for Parts Of Speech."""
 
+    objects = ParentManager()
```

Comment:
Doing this overrides the custom manager Nolan added to handle permissions, but since PartOfSpeech is not access-controlled I think it's alright

Reply:
Now that I think of it, it can override the Permissions manager (I'll double check in the PartsOfSpeech API branch, once this gets merged).
My thinking of why it can override is that both managers are being assigned to the objects attribute. One way to avoid this would be to assign one of these managers to a specific attribute, for e.g. if we are assigning the permissions manager (coz of it being explicitly used in a queryset), to say 'permitted' attribute (or some more suitable name), then in the FVPermissionViewSetMixin, it can be called by `self.queryset.model.permitted.get_viewable_for_user(request.user) `.

@NolanVH your thoughts on this ?

Reply:
In this case, it seems like we don't need both but if we do could the ParentManager not just inherit from PermissionsManager? That way we get the functionality from both and we also keep the `self.queryset.model.object.get_viewable_for_user(request.user)` intact.